contracts/openzeppelin/token/ERC20/IERC20.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 // OpenZeppelin Contracts (last updated v4.6.0) (token/ERC20/IERC20.sol)
 
-pragma solidity ^0.8.0;
```

Comment:
Do we want to modify the OZ contracts too? I remember it was mentioned that we shouldn't modify them, or is that ok if we only modify the Solidity version? @StanislavBreadless @vladbochok

Reply:
@koloz193 also there is a missing space in the OZ contracts solidity versions: it is `pragma solidity0.8.20;` instead of `pragma solidity 0.8.20;`

Reply:
> or is that ok if we only change the Solidity version?

It is a separate thing and I would rather not update OZ now